src/oc_erchef/apps/chef_db/src/chef_pgsql_collector.erl

```diff
@@ -0,0 +1,92 @@
+-module(chef_pgsql_collector).
+
+-export([deregister_cleanup/1,
+         collect_mf/2,
+         collect_metrics/2]).
+
+-import(prometheus_model_helpers, [create_mf/5,
+                                   label_pairs/1,
+                                   gauge_metrics/1,
+                                   gauge_metric/1,
+                                   gauge_metric/2,
+                                   counter_metric/1,
+                                   counter_metric/2]).
+
+-behaviour(prometheus_collector).
+
+-define(METRICS, [
+                  {<<"seq_scan">>, {pg_stat_seq_scan, counter, "Number of sequential scans initiated", fun erlang:binary_to_integer/1}},
+                  {<<"seq_tup_read">>, {pg_stat_seq_tup_read, counter, "Number of live rows fetched by sequential scans", fun erlang:binary_to_integer/1}},
+                  {<<"idx_scan">>, {pg_stat_idx_scan, counter, "Number of index scans initiated", fun erlang:binary_to_integer/1}},
+                  {<<"idx_tup_fetch">>, {pg_stat_tup_fetch, counter, "Number of live rows fetched by index scans", fun erlang:binary_to_integer/1}},
+                  {<<"n_tup_ins">>, {pg_stat_n_tup_ins, counter, "Number of rows inserted", fun erlang:binary_to_integer/1}},
```

Comment:
We should make it clear in these descriptions that these are sums across all tables perhaps.

Reply:
:+1: